packages/frontend/src/commons/chains/gnosis.ts

```diff
@@ -0,0 +1,66 @@
+import { SupportedDex } from "@metrom-xyz/sdk";
+import { SupportedChain, ADDRESS } from "@metrom-xyz/contracts";
+import { type ChainData } from "..";
+import { GnosisLogo } from "@/src/assets/logos/chains/gnosis";
+import { gnosis } from "viem/chains";
+import { SwaprLogo } from "@/src/assets/logos/dexes/swapr";
+
+export const gnosisData: ChainData = {
+    name: gnosis.name,
+    metromContract: ADDRESS[SupportedChain.Taiko],
```

Comment:
Wrong `SupportedChain` here.

Reply:
Addressed in 261709c46d2d09cc3688acc348b577ab5d969315